build.include

```diff
@@ -94,16 +94,15 @@ buildImages() {
 }
 
 publishImages() {
-  echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  REGISTRY="quay.io"
   for image in "${PUBLISH_IMAGES_LIST[@]}"
   do
     echo "Publishing image ${image}..."
+    docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${IMAGE_TAG}"
+    echo y | docker push "${REGISTRY}/${image}:${IMAGE_TAG}"
     if [[ -n "${THEIA_DOCKER_IMAGE_VERSION}" ]]; then
-      docker tag "${image}:${IMAGE_TAG}" "${image}:${THEIA_DOCKER_IMAGE_VERSION}"
```

Comment:
please include this fix here https://github.com/eclipse/che-theia/pull/978 so we can get both nightly/next AND a short-sha-tagged release too.

Reply:
sure, will do
thanks for reminding me!

Reply:
no problem! it's a new change, so figured it would be good to get it in here too. See above re: secrets -- hope that unblocks you!